lib/oidcc/plug/authorization_callback.ex

```diff
@@ -220,17 +217,46 @@ defmodule Oidcc.Plug.AuthorizationCallback do
     |> put_private(__MODULE__, result)
   end
 
+  defp get_client_context(conn, opts) do
+    if client_store = Keyword.get(opts, :client_store) do
```

Comment:
Can we do some checking that the options are exclusive? (Prevent misunderstandings...)